test/integration_test/test-deploy.sh

```diff
@@ -7,6 +7,7 @@ test_image_name="stork_test:latest"
 remote_config_path=""
 run_cluster_domain_test=false
 environment_variables=""
+storage_provisioner=""
```

Comment:
Initialize this to portworx, so you don't need the `if/else` below

Reply:
Fixed.